src/app/pipes/center-truncate.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'centerTruncate'
```

Comment:
truncateMiddleOfString or truncateMiddleOfText

To clearly convey the action of truncating the middle section of the string/text with the preservation of both ends